src/components/Book.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Students are not permitted to use ESLint-disabler. I strongly encourage you to remove this line and address the indicated error.